Sources/SPMBuildCore/BinaryTarget+Extensions.swift

```diff
@@ -93,7 +93,7 @@ fileprivate extension Triple.OS {
     /// Returns a representation of the receiver that can be compared with platform strings declared in an XCFramework.
     var asXCFrameworkPlatformString: String? {
         switch self {
-        case .darwin, .linux, .wasi, .windows, .openbsd:
```

Comment:
This is not correct, XCFrameworks support all Apple platforms. The strings are:

- macos
- ios
- ios-simulator
- ios-macabi
- ios-maccatalyst
- tvos
- tvos-simulator
- watchos
- watchos-simulator
- driverkit